.pre-commit-hooks.yaml

```
@@ -30,13 +37,39 @@
   types: [text]
   stages: [commit, push, manual]
 
-- id: trailing-whitespace
-  name: Trim Trailing Whitespace
-  description: This hook trims trailing whitespace.
-  entry: trailing-whitespace-fixer
+- id: flake8
+  name: flake8
+  description: flake8 - Python Code Formatter
   language: python
+  entry: flake8
+  files: \.(py)$
```

Shoudn't this be
```suggestion
  files: \.py$
```
?

---

Doesn't change much from a regex point of view. But it can be what you suggest I think.